frontend/src/features/projects/dispose/components/columns.tsx

```diff
@@ -32,12 +32,18 @@ const EditableMoneyCell = (cellInfo: any) => {
 
 const EditableClassificationCell = (cellInfo: any) => {
   const classifications = useCodeLookups().getOptionsByType('PropertyClassification');
+  const filteredClassifications = classifications.filter((x: any) => {
+    if (x.label === 'Surplus Active' || x.label === 'Surplus Encumbered') {
+      return x;
+    }
+    return null;
+  });
   const context = useFormikContext();
```

Comment:
Please change this to return a proper boolean value. This works because x is "truthy" and null is "falsy" but can lead to bugs.

Basically, for `.filter()` if the callback function returns true, the current element will be in the resulting array. If it returns false, it won’t be. So there's no need to return the element here, just true or false.

This could be re-written as:

```ts
 const filteredClassifications = classifications.filter((x: any) => x?.label === 'Surplus Active' || x?.label === 'Surplus Encumbered');

```

Reply:
Ahh I knew it didn't feel right, thanks for the help